api/src/main/java/org/pmiops/workbench/api/CohortsController.java

```diff
@@ -74,7 +74,7 @@ public Cohort apply(DbCohort cohort) {
                   .name(cohort.getName())
                   .type(cohort.getType());
           if (cohort.getCreator() != null) {
-            result.setCreator(cohort.getCreator().getEmail());
+            result.setCreator(cohort.getCreator().getUsername());
```

Comment:
@freemabd FYI the `email` entity field is being replaced but we're keeping the table schema as is. So it's just a replacement for that column. Take care with manual sql queries. I found one place I had to fix it by hand.

Reply:
Thanks for info.